caliban_toolbox/utils/plot_utils.py

```diff
@@ -132,51 +134,46 @@ def overlay_crop_overlap(img_crop, row_starts, row_ends, col_starts, col_ends):
     return img_crop
 
 
-def set_channel_colors(combined_xr, plot_colors):
+def set_channel_colors(channel_data, plot_colors):
     """Modifies the order of image channels so they're displayed with appropriate color in caliban
 
     Args:
-        combined_xr: xarray containing channels and labels
-        plot_colors: array containing the color of each channel, in order of the current channels
+        channel_data: xarray containing channels
+        plot_colors: array containing the desired color for each channel
```

Comment:
Where do we handle the colors for the labels now that they are split out from the xarray data?

Reply:
Caliban has a color map that determines how label colors are handled. The channel colors are determined by their index in the RGBCMY array that is used for plotting. So the above function puts the channel in right order for the desired color, but leaves it up to caliban to set the color for a specific label.